Debian-specific config for pwauth. Also use proper style comments for C code.

Author: Hai Zaar <haizaar@haizaar.com>

--- a/config.h
+++ b/config.h
@@ -121,22 +121,22 @@
 
    /* LOW-LEVEL OPTIONS */
 
-/* #define SHADOW_NONE		/**/
-/* #define SHADOW_BSD		/* FreeBSD, NetBSD, OpenBSD, BSDI, OS X */
-#define SHADOW_SUN		/* Linux, Solaris, IRIX */
-/* #define SHADOW_JFH		/**/
-/* #define SHADOW_MDW		/**/
-/* #define SHADOW_AIX		/* AIX (see also AUTHENTICATE_AIX) */
-/* #define SHADOW_HPUX		/* HPUX ? */
+/* #define SHADOW_NONE		*/
+/* #define SHADOW_BSD		 FreeBSD, NetBSD, OpenBSD, BSDI, OS X */
+/* #define SHADOW_SUN		 Linux, Solaris, IRIX */
+/* #define SHADOW_JFH		*/
+/* #define SHADOW_MDW		*/
+/* #define SHADOW_AIX		 AIX (see also AUTHENTICATE_AIX) */
+/* #define SHADOW_HPUX		 HPUX ? */
 
    /* HIGH-LEVEL OPTIONS */
 
-/* #define PAM			/* Linux PAM or OpenPAM */
-/* #define PAM_OLD_OS_X		/* PAM on OS X version 10.5 or older */
-/* #define PAM_SOLARIS		/* PAM on Solaris other than 2.6 */
-/* #define PAM_SOLARIS_26	/* PAM on Solaris 2.6 */
-/* #define LOGIN_CONF_OPENBSD	/* login.conf on OpenBSD */
-/* #define AUTHENTICATE_AIX	/* AIX authenticate() function */
+#define PAM			/* Linux PAM or OpenPAM */
+/* #define PAM_OLD_OS_X		 PAM on OS X version 10.5 or older */
+/* #define PAM_SOLARIS		 PAM on Solaris other than 2.6 */
+/* #define PAM_SOLARIS_26	 PAM on Solaris 2.6 */
+/* #define LOGIN_CONF_OPENBSD	 login.conf on OpenBSD */
+/* #define AUTHENTICATE_AIX	 AIX authenticate() function */
 
 
 /* There is also limited support for three failure logging systems (the
@@ -193,13 +193,13 @@
  * reports and resets the failure count.
  */
 
-/* #define FAILLOG_JFH				/**/
-/* #define FAILLOG_OPENBSD			/**/
-/* #define FAILLOG_PWAUTH			/**/
-
-/* #define PATH_FAILLOG "/var/log/faillog"	/**/
-/* #define MAX_FAIL_COUNT 40			/**/
-/* #define RESET_FAIL_COUNT			/**/
+/* #define FAILLOG_JFH				*/
+/* #define FAILLOG_OPENBSD			*/
+/* #define FAILLOG_PWAUTH			*/
+
+/* #define PATH_FAILLOG "/var/log/faillog"	*/
+/* #define MAX_FAIL_COUNT 40			*/
+/* #define RESET_FAIL_COUNT			*/
 
 
 /* If UNIX_LASTLOG is defined, the program will update the lastlog entry so
@@ -213,8 +213,8 @@
  * uid numbers.
  */
 
-#define UNIX_LASTLOG		/**/
-#define HAVE_LASTLOG_H		/**/
+/* #define UNIX_LASTLOG		*/
+/* #define HAVE_LASTLOG_H		*/
 
 
 /* If NOLOGIN_FILE is defined to the full path name of a file, then the
@@ -227,8 +227,8 @@
  * right with PAM.
  */
 
-#define NOLOGIN_FILE "/etc/nologin"	/**/
-#define MIN_NOLOGIN_UID 1		/**/
+/* #define NOLOGIN_FILE "/etc/nologin"	*/
+/* #define MIN_NOLOGIN_UID 1		*/
 
 
 /* Defining CHECK_LOGIN_EXPIRATION and CHECK_PASSWORD_EXPIRATION causes
@@ -240,8 +240,8 @@
  * system to test on, I haven't bothered.
  */
 
-#define CHECK_LOGIN_EXPIRATION		/**/
-#define CHECK_PASSWORD_EXPIRATION	/**/
+/* #define CHECK_LOGIN_EXPIRATION		*/
+/* #define CHECK_PASSWORD_EXPIRATION	*/
 
 
 /* It is generally sensible to restrict what users can run pwauth.  Though
@@ -279,7 +279,7 @@
  * to change the uid list.
  */
 
-#define SERVER_UIDS 30		/* user "wwwrun" on the author's system */
+/*#define SERVER_UIDS 30  */		/* user "wwwrun" on the author's system */
 
 
 /* If MIN_UNIX_UID is defined to an integer, logins with uid numbers less than
@@ -291,7 +291,7 @@
  * given value will be accepted).
  */
 
-#define MIN_UNIX_UID 500	/**/
+#define MIN_UNIX_UID 100	/**/
 
 
 /* If IGNORE_CASE is defined, the login given is checked in two different
@@ -300,7 +300,7 @@
  * This ignores the case of the login name only, not the password.
  */
 
-/* #define IGNORE_CASE             /**/
+/* #define IGNORE_CASE             */
 
 
 /* If DOMAIN_AWARE is enabled, then we we check login names to see if they
@@ -309,7 +309,7 @@
  * to login names formed like "domain\username".
  */
 
-/* #define DOMAIN_AWARE            /**/
+/* #define DOMAIN_AWARE            */
 
 
 /* On failed authentications, pwauth will sleep for SLEEP_TIME seconds, using
@@ -320,7 +320,7 @@
  * SLEEP_LOCK.  SLEEP_TIME defaults to 2 seconds if not defined.
  */
 
-#define SLEEP_LOCK "/var/run/pwauth.lock"
+/* #define SLEEP_LOCK "/var/run/pwauth.lock" */
 
 
 /* If ENV_METHOD is defined, pwauth expects mod_auth_external to be configured
@@ -330,7 +330,7 @@
  * This is insecure on some versions of Unixes, but might be a bit faster.
  */
 
-/* #define ENV_METHOD		/**/
+/* #define ENV_METHOD		*/
 
 
 /* If /usr/include/paths.h exists define this.  Obviously I need to autoconfig
